permissions/plugin-kubernetes-credentials-provider.yml

```diff
@@ -0,0 +1,6 @@
+---
+name: "kubernetes-credentials-provider"
+paths:
+- "com/cloudbees/jenkins/plugins/k8sCredentialProvider"
```

Comment:
urgh, using not only lowercase is looking for pain IMO... I guess it's too late anyway

Reply:
not tool late there is no release.  
And well even after I can change it as no data is persisted, nor does this constitute a public API.

that said `k8s_credential_provider` would be the convention that is not really any better.

Reply:
do we really even need such a specific groupId? Why not just `com.cloudbees.jenkins.plugins` like Daniel is saying too below?